pandas/util/testing.py

```diff
@@ -2121,6 +2121,20 @@ def __init__(self, *args, **kwargs):
         dict.__init__(self, *args, **kwargs)
 
 
+class TestNonDictMapping(abc.Mapping):
```

Comment:
Can you make this a fixture in `pandas/tests/series/conftest.py` instead of putting in `pandas.util.testing`? With the latter sometimes things can leak into our API and definitely wouldn't want that here